app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/HorizontalSnapScrollView.java

```diff
@@ -186,32 +187,46 @@ public boolean onTouch(View v, MotionEvent event) {
         }
     }
 
-    // FIXME: Landscape patch to expand a column to full width if there is space available
-    private static int calcMaxCols(Resources res, int availPixels, int columnsCount) {
-        int maxCols = res.getInteger(R.integer.max_cols);
-        // TODO: The next line is the relevant monkey patch
-        maxCols = Math.min(columnsCount, maxCols);
-        // TODO: The previous line is the relevant monkey patch
-        int minWidthDip = res.getInteger(R.integer.min_width_dip);
-        float scale = res.getDisplayMetrics().density;
-        MyApp.LogDebug(LOG_TAG, "calcMaxCols: avail " + availPixels + " min width dip " + minWidthDip);
-        int dip;
-        do {
-            dip = (int) ((((float) availPixels) / maxCols) / scale);
-            MyApp.LogDebug(LOG_TAG, "calcMaxCols: " + dip + " on " + maxCols + " cols.");
-            maxCols--;
-        } while (dip < minWidthDip && maxCols > 0);
-        return maxCols + 1;
+    /**
+     * Calculates the number of columns to display at a time based on the physical dimensions and
```

Comment:
I would like to provide some prose to protect the average code traveler from requiring to read the implementation. Since the parameters are now part of the signature I have hope that the JavaDoc will be updated, too. :crossed_fingers: